pages/common/encfs.md

```diff
@@ -0,0 +1,24 @@
+# encfs
+
+> Encrypted virtual filesystem.
+> More information: <https://github.com/vgough/encfs>.
+
+- Initialize or mount an encrypted filesystem:
+
+`encfs {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Initialize an encrypted filesystem with standard settings:
+
+`encfs --standard {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Run encfs in foreground instead of spawning a daemon:
+
+`encfs -f {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
```

Comment:
Is there a long option for `-f`?  No problem if there isn't.

Reply:
Not that I am aware of, no mention of longer alternative in manpage either.